### ISSUE TITLE
Single song being split into two separate files

### ISSUE THREAD

**Issue by FreshDumb** [jwallet/spy-spotify#226]
Overview:
Happens very rarely, but sometimes a single song is split into separate files.

How to reproduce:
1. Try to record the following Song:
Joris Voorn - Messiah (ft. HÆLOS)

2. The track is split into separate files. This happened consistently for me.

**Track used**
https://open.spotify.com/track/6lZwegV6oPqgNtvGDXWrt6?si=LC3_bJjQTEehtfC4rvgHxA

**Screenshots**
Settings 1:
![image](https://user-images.githubusercontent.com/22946660/80931263-e6914380-8db8-11ea-8437-5ecf735ac218.png)

Settings 2:
![image](https://user-images.githubusercontent.com/22946660/80931274-fb6dd700-8db8-11ea-93bf-7c322aafcd81.png)

**Comment by jwallet** [comment]
i did not work on your issue yet, but this is may be affected by `Delay next recording` that uses the length of the song that the app got from lastfm.api, if it is wrong it might changes track before it ends. You can try SpotifyAPI instead https://jwallet.github.io/spy-spotify/faq.html#media-tags-not-found

**Comment by FreshDumb** [comment]
Ah, that makes sense, although the Song length of this Song is the same in lastfm as far as i can tell.
I'll try the SpotifyAPI and update you.
Also that kind of confuses me, since `delay next recording` sounds like it should do the opposite or am i misunderstanding the name of the option?

Does .wav recording use lastfm or spotify api? Just asking, because the choice disappears when selecting .wav format.

On a side note, great work with this tool. Has been really useful. :)

**Comment by jwallet** [comment]
> Does .wav recording use lastfm or spotify api? Just asking, because the choice disappears when selecting .wav format.

No api, wave don't include id3 tags

**Comment by FreshDumb** [comment]
Ah allright.

Just tested with the Spotify API and everthing is fine there. 👍 
Does changing the API change anything apart from where id3 tags are gotten from?

**Comment by jwallet** [comment]
nothing more at the moment. spytify used to work with spotify open api but they shut it down a while ago and I rewrote the app to use lastfm api because this one is still open to anonymous queries. It is possible to use Spotify API for status as well, but it requires every user to create their own token and grant access to it, a bit overhead for someone that records main stream tracks, main streams track are available on lastfm.api. Right now I just added spotify-api id3tags, but there are open issue for status #137 and login #120
